tests/mat.rs

```diff
@@ -23,30 +26,51 @@ fn mat_for_rows_and_cols() {
     assert_eq!(120000, mat.total().unwrap());
 }
 
+
 #[test]
-fn mat_at() {
```

Comment:
I broke this test up into a couple smaller tests and also switched the Mat default scalar to `1.23` to help rule-out coincidental uninitialized memory zeros.